### ISSUE TITLE
Update docs for building with oneDNN execution provider GPU support. OpenCL SDK deprecated.

### ISSUE THREAD

**Issue by prikmm** [microsoft/onnxruntime#11414]
**Describe the bug**
Currently, the [build](https://onnxruntime.ai/docs/execution-providers/oneDNN-ExecutionProvider.html#build) docs state that inorder to build ONNXRuntime with oneDNN execution provider with GPU support, we need to install [OpenCL SDK](https://www.intel.com/content/www/us/en/developer/tools/opencl-sdk/overview.html). OpenCL SDK has been deprecated and its last update was in Oct, 2020.
 For more info checkout: https://www.intel.com/content/dam/develop/external/us/en/documents/opencl-sdk-2020-u3-release-notes.pdf

**Urgency**
No such timeline, I am working on a personal project and would like to compare performance of different execution providers.

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 20.04
- ONNX Runtime installed from (source or binary): source
- ONNX Runtime version: 1.11.0
- Python version: 3.8
- Visual Studio version (if applicable): 
- GCC/Compiler version (if compiling from source): 9.4.0
- CUDA/cuDNN version: NA
- GPU model and memory: NA

**Expected behavior**
Updated documentation showing new steps for building ORT with oneDNN EP with GPU support if possible would be great. If the oneDNN EP is completely dependent on OpenCL SDK and cannot be compiled using OpenCL Runtime, then, a deprecation warning regarding such would be good.

**Comment by jywu-msft** [comment]
@georgen117 can you comment on this?

**Comment by jeyblu** [comment]
OpenCL SDK can still be downloaded and used to build ORT with oneDNN EP. Scroll down to the middle of the [OpenCL SDK](https://www.intel.com/content/www/us/en/developer/tools/opencl-sdk/overview.html) page for the download link to the [SDK](https://www.intel.com/content/www/us/en/developer/tools/opencl-sdk/choose-download.html) and [runtime](https://www.intel.com/content/www/us/en/developer/articles/tool/opencl-drivers.html).

**Comment by prikmm** [comment]
@jeyblu I would like to install ORT with oneDNN EP with GPU support in docker. I searched a lot but was not able to find a way to directly download the SDK on inside a docker container using `wget` or `curl`. Also, SDK required licensing when I installed it on my local computer. Is there a way to overcome this issue? I think Intel OpenCL C++ compiler which cannot be used without licensing is necessary for building, right?

Please correct me if I am wrong anywhere.

**Comment by jeyblu** [comment]
To get the SDK, I think it first asks you to register and sign in, so wget or curl may not work. You should be able to register and download the OpenCL SDK from the [download](https://www.intel.com/content/www/us/en/developer/tools/opencl-sdk/choose-download.html) page without having to pay. The paid version is for additional tools and support.

**Comment by prikmm** [comment]
@jeyblu I knew about downloading the SDK, but wanted to clarify about `wget` and `curl`. 

I will still keep the issue open as OpenCL SDK, though currently available for download, as reached it end of lifecycle. So, I think mentioning this or an alternative method in the docs is necessary.

**Comment by jeyblu** [comment]
Makes sense to mention that there will be no new releases of OpenCL SDK. We're investigating an alternative to the OpenCL SDK.

**Comment by chethanpk** [comment]
We have updated the docs to show an alternate path to build: using Khronos OpenCL SDK. I am not sure if there us anything else needed from this issue.
